corehq/apps/api/resources/v0_5.py

```diff
@@ -1423,5 +1423,5 @@ def prepend_urls(self):
         # We're overriding the default "list" view to redirect to "detail" view since
         # we already know the user through OAuth.
         return [
-            url(r"^(?P<resource_name>%s)/$" % self._meta.resource_name, self.wrap_view('dispatch_detail')),
+            url(r"^$", self.wrap_view('dispatch_detail'), name='api_dispatch_detail'),
```

Comment:
I tried to understand the impact of the linked PR on to this one, but could not really figure it out.

The resource name [now](https://github.com/dimagi/commcare-hq/pull/35048/files#diff-4cc305529a1f35d435b4f08f6c4b775e1d1eb18f9664ae592e8dff953fdb25c4) comes before the version?
Or may be the question is,
What was the URL earlier and what is it now?

Reply:
I moved the `resource_name` out of this portion of the URL and up [a level higher](https://github.com/dimagi/commcare-hq/pull/35048/files#diff-91c39ba9e6f0b8b63caaeb3f9f8b4b59d01121fa7a3f21b4e4b15d4c08a2fb6fR104).

The conflict between my PR and Charl's meant that `(?P<resource_name>%s)` would've appeared in the URL twice.

Reply:
Thsnks @esoergel 

Do users need to update any of the urls they might be using like we do here https://github.com/dimagi/commcare-analytics/blob/master/hq_superset/hq_url.py?

Reply:
Nope, all the existing URLs should continue to function as before.  Though if you guys like, you could make this the first API _only_ available at the new URL (assuming you can be confident that that's the only place referencing this API)

Reply:
How about [this](https://github.com/dimagi/commcare-hq/pull/35312/commits/76d4f6dcb8cf4c85d193f5e53c0de8811935bc56)?

Reply:
Solid, thank you @Charl1996 @esoergel